### ISSUE TITLE
Save a rotated image

### ISSUE THREAD

**Issue by Lucia2912** [openseadragon/openseadragon#2269]
I need to save an image after rotating it with the position in which it is left
Is there any option or property that allows you to save an image after rotating it?

**Comment by iangilman** [comment]
You mean save the pixels of the image? The easiest thing to do is copy the contents of the canvas. That gives you exactly what you see in the viewport at the moment. 

Or do you mean remember where you are rotated so you can load the viewer up that way again later?

**Comment by Lucia2912** [comment]
Yes, I mean remember where you are rotated so you can load the viewer up that way again later.
For example having a vertical image, it is rotated to horizontal so in the next few times you open the viewer the image is seen horizontally.
It is possible?

**Comment by iangilman** [comment]
Sure. You'll have to save the value yourself (either on the server, or locally using local storage or something). If you're rotating using the viewport, you can get the current value with `viewer.viewport.getRotation()`. Store that somewhere and next time you load, you can use `viewer.viewport.setRotation` with that value.
